source/elements/tags/Alert/Alert.css

```diff
@@ -0,0 +1,47 @@
+.Alert {
+  background-color: #f1f1f1;
```

Comment:
It would be helpful to create color variables for these background-colors in the 'variables/colors.css' file. That way they are more easily targeted when a project gets up and running. They would fit well under the `/* general colors */` comment.